src/index.js

```diff
@@ -0,0 +1,38 @@
+import './style.css';
+
+const ul = document.getElementById('list');
+const tasks = [
+  {
+    index: 1,
+    isCompleted: false,
+    description: 'do coding challenge',
+  },
+  {
+    index: 2,
+    isCompleted: false,
+    description: 'Read the Bible',
+  },
+  {
+    index: 3,
+    isCompleted: false,
+    description: 'Read emails',
+  },
+];
+
+function displaylist() {
+  tasks.forEach((task) => {
+    const li = document.createElement('li');
+    const text = `<div> <input class='form-check-input me-1' type='checkbox' value='' aria-label='...'>
+    ${task.description}</div>`;
+    li.classList.add('list-group-item');
+    li.innerHTML = text;
+    ul.appendChild(li);
+  });
```

Comment:
- [ Optional] As per the learning objective we have to use proper ES6 syntax. Try to use the arrow function as per the ES6 guidelines.